compositor_api/src/types.rs

```diff
@@ -34,6 +34,7 @@ pub use component::View;
 pub use component::WebView;
 
 pub use register_input::Mp4;
+pub use register_output::Mp4Output;
```

Comment:
If this is a user-facing value, use Mp4 too. If it not user-facing, I would rename input mp4 for consistency